api/deploy.go

```diff
@@ -139,6 +139,10 @@ func (d *serviceDeployer) deploy(r io.Reader) (*service.Service, *importer.Valid
 	if validationErr != nil {
 		return nil, validationErr, nil
 	}
+
+	if err := s.CreateVolumes(); err != nil {
```

Comment:
Why do we need to create volumes here, they are dynamically created when we start a service so I don't see the benefits of adding this here (and the whole service/volumes file)

Reply:
I added to set some labels on the volume but after I removed the labels. I'm deleting the creation of volumes part.